src/utils/tokenHelper.js

```diff
@@ -44,18 +44,16 @@ function getTokenBySymbol(symbol) {
 }
 
 const getValidSymbols = () => {
-  if (validSymbols.length) {
-    return validSymbols;
+  if (validSymbols.size) {
+    return Array.from(validSymbols);
   }
   getWhiteListTokens().forEach(token => {
-    if (!validSymbols.includes(token.symbol)) {
-      validSymbols.push(token.symbol);
-    }
-    if (token.rateEqSymbol && !validSymbols.includes(token.rateEqSymbol)) {
-      validSymbols.push(token.rateEqSymbol);
+    validSymbols.add(token.symbol);
+    if (token.rateEqSymbol) {
+      validSymbols.add(token.rateEqSymbol);
     }
   });
-  return validSymbols;
+  return Array.from(validSymbols);
 };
 
 const isSymbolInTokenWhitelist = symbol => {
```

Comment:
Please update this method to use directly the `validSymbols` set instead of converting it once to array and then calling the `includes` method of array.

Reply:
@aminlatifi 
You know we use `getValidSymbols()` somewhere in the codes, So I preferred to not change the signature of `getValidSymbols()`
https://github.com/Giveth/feathers-giveth/blob/develop/src/services/conversionRates/conversionRates.service.js#L43

Is it worth changing it ?(and we should change some test cases as well)

Reply:
I meant `isSymbolInTokenWhitelist` method

Reply:
I put the comment on that line, but I don't know why GitHub shows it in this window!

Reply:
`isSymbolInTokenWhitelist()` use `getValidSymbols()` inside of that, so if I want to change it, I had to change `getValidSymbols()` output and it would have some side effects

Reply:
I meant writing something like this
```suggestion
const isSymbolInTokenWhitelist = symbol => {
  return validSymbols.has(symbol);
};
```

But there is a point, the validSymbols may not be initialized! to solve it we can initialize it as application starts since the config is changed and it will be done once. Initializing at start up also reduces one comparison from `getValidSymbols` function